review/models.py

```diff
@@ -0,0 +1,91 @@
+from django.db import models
+from user.models import User
+from item.models import Item
+from store.models import Store
+
+class ItemReview(models.Model):
+	user				= models.ForeignKey('User', on_delete=models.SET_NULL, null=True)
+	item				= models.ForeignKey('Item', on_delete=models.SET_NULL, null=True)
+	packaging_rating	= models.ForeignKey('Ratings', on_delete=models.SET_NULL, null=True)
```

Comment:
어차피 한가지 선택을 저장하게 되는거라면 연결 성격상 OneToOneField를 추천드립니다.
```suggestion
	packaging_rating	= models.ForeignKey('Rating', on_delete=models.SET_NULL, null=True)
```
아래의 두 rating도 부탁드립니다.